README.md

```diff
@@ -245,17 +245,12 @@ http://<key>@localhost:3001/<id>
 
 ### Release Management
 
-We use [craft](https://github.com/getsentry/craft) to release new versions.
+We use GitHub actions to release new versions.
 There are two separate projects to publish:
 
-- **Relay binary** is released from the root folder. Run `craft prepare` and
-  `craft publish` in that directory to create a release build and publish it,
-  respectively. We use [Calendar Versioning](https://calver.org/) and coordinate
-  releases with Sentry.
+- **Relay binary** is released with the ["Release" action](https://github.com/getsentry/relay/actions/workflows/release_binary.yml). Make sure that [`CHANGELOG.md`](./CHANGELOG.md) is up-to-date before running the action. Hit "Run workflow" and choose a new version. We use [Calendar Versioning](https://calver.org/) and coordinate releases with Sentry.
```

Comment:
Could you mention explicitly that Relay isn't supposed to be released manually here, but in the federated release? It's still useful to describe the release action and everything that's written here, but I'd put slightly more emphasis on the calver part and automated monthly releases.

Also: 

```suggestion
- **Relay binary** is released with the ["Release" action](https://github.com/getsentry/relay/actions/workflows/release_binary.yml). Make sure that [`CHANGELOG.md`](./CHANGELOG.md) is up-to-date before running the action. Press "Run workflow" and choose a new version. We use [Calendar Versioning](https://calver.org/) and coordinate releases with Sentry.
```